### PR TITLE
Update to epoch loss calculation

### PR DESCRIPTION
**Pull request by Fabio-83**
## Related issues

Loss calculation in some of the examples was higher for the validation set than the training set

## Description

fix in the validation loss calculation and update to the hook call for the training and validation runner metrics

### CODE REVIEW

**Review by diego-plan9 — APPROVED** [review]
Thanks @Fabio-83 !
